package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "jade-data-repo-ui",
-  "version": "0.176.0",
+  "version": "0.181.0",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "jade-data-repo-ui",
-      "version": "0.176.0",
+      "version": "0.181.0",
```

Comment:
If you get the latest code from develop, we shouldn't need this change, right? 
I'm just worried there might be some conflicts when merging this change since we automatically bump the version.

Reply:
Ok, let me revert this back!